src/main/java/com/charu/Operations.java

```diff
@@ -0,0 +1,77 @@
+package com.charu;
+import java.util.Scanner;
+
+import java.util.*;
+public class Operations {
+    Tasks task ;
+    Scanner myObj = new Scanner(System.in);
+    ArrayList<Tasks> lists = new ArrayList<Tasks>();
+    int counter;
+    public void add(){
+     if(!lists.isEmpty()){
+    counter+=lists.get(lists.size()-1).getId();
+     }
+     else{
+        counter=1;
+     }
+    System.out.println("Add name");
+    String name = myObj.next();
+    task = new Tasks(counter, name, "inprogress");
+    lists.add(task);
+    System.out.println(lists);
```

Comment:
Call the method to `fetchTasks()` where you can display the data in a tabular format

Reply:
addressed